src/Basement/CMakeLists.txt

```diff
@@ -1,12 +1,6 @@
 project(Basement Fortran)
 
-cmake_minimum_required(VERSION 3.5.1)
-
-cmake_policy(SET CMP0012 NEW) # if() recognizes boolean constants
-
-if(POLICY CMP0087)
-  cmake_policy(SET CMP0087 NEW) # install(CODE) and install(SCRIPT) support generator expressions.
-endif()
```

Comment:
And remove these old expliclit ones.

CMP0012 was aded in 2.8.0,
CMP0048 in 3.0
CMP0054 in 3.1